pkg/daemon/daemon.go

```diff
@@ -1278,15 +1312,16 @@ func ValidPath(path string) bool {
 // senseAndLoadOnceFrom gets a hold of the content for supported onceFrom configurations,
 // parses to verify the type, and returns back the genericInterface, the type description,
 // if it was local or remote, and error.
-func (dn *Daemon) senseAndLoadOnceFrom() (interface{}, onceFromOrigin, error) {
+func (dn *Daemon) senseAndLoadOnceFrom(onceFrom string) (interface{}, onceFromOrigin, error) {
```

Comment:
Just wondering, since this function is not utilizing Daemon struct anymore, is there a reason to keep it part of Daemon ?

Reply:
not really, we can remove the object reference to Daemon as a cleanup